week09/movie/src/components/Detail.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "styled-components";
+
+const Title = styled.div`
+    font-size: 40px;
+    margin: 10px;
+`;
+const ImgAndInfo = styled.div`
+    display: flex;
+`;
+const Info = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin: 10px;
+`;
+const Desc = styled.div`
+    font: 10px;
+    margin: 10px;
+`;
+
+const Div = styled.div`
+    font-size: 23px;
+    margin: 10px;
+`;
+
+const Detail = ({ info }) => {
+    const title = info.title;
+    const img = info.medium_cover_image;
+    const year = info.year;
+    const rating = info.rating;
+    const runtime = info.runtime;
+    const genres = info.genres;
+    const description = info.description_full;
+
+    return (
+        <div>
+            <Title>{title}</Title>
+            <ImgAndInfo>
+                <img src={img} style={{ marginLeft: "10px" }} />
```

Comment:
- [x] 1. 이미지 대체텍스트
```suggestion
                <img src={img} style={{ marginLeft: "10px" }} alt="Movie Poster" />
```
alt 속성으로 대체 텍스트를 포함하여 ESLint 오류를 제거합니다.